### PR TITLE
feat: add alias prompt to authorize a devhub

### PR DESCRIPTION
**Pull request by CristiCanizales**
### What does this PR do?
Enable the user to define an alias when the command "Authorize a DevHub" runs

### What issues does this PR fix or reference?
@W-11853620@
[#2278](https://github.com/forcedotcom/salesforcedx-vscode/issues/2278)

### Functionality Before
image

### Functionality After
image

### CODE REVIEW

**Comment by CristiCanizales** [comment]
Hello @andruts ! Could you please sign the cla? Thank you in advance!

**Comment by andruts** [comment]
Hello @CristiCanizales, I already signed the CLA:
image

I just added my personal email to my GitHub account. Hope that works.

File: packages/salesforcedx-vscode-core/src/commands/auth/forceAuthDevHub.ts [hunk 1]
```diff
@@ -58,17 +60,20 @@ export class ForceAuthDevHubContainerExecutor extends ForceAuthWebLoginContainer
   }
 }
 
+export const DEFAULT_ALIAS = 'vscodeOrg';
```
Comment (gbockus-sf) [inline packages/salesforcedx-vscode-core/src/commands/auth/forceAuthDevHub.ts:63]: can you update this to use: https://github.com/forcedotcom/salesforcedx-vscode/blob/96fe184c8440c206ddb4bc004387eb17c454db77/packages/salesforcedx-vscode-core/src/commands/auth/authParamsGatherer.ts#L18

**Review by gbockus-sf — APPROVED** [review]
One request to update a constant that is already defined elsewhere.

**Comment by CristiCanizales** [comment]
Thanks for contributing @andruts !😁

**Pull request merged**